src/graph/ExecutionContext.cpp

```diff
@@ -15,6 +15,10 @@ ExecutionContext::~ExecutionContext() {
         sm_ = nullptr;
     }
 
+    if (nullptr != gflagsManager_) {
+        gflagsManager_ = nullptr;
```

Comment:
maybe here can set nullptr directly.

Reply:
I just keep the same style as above - -.